napari/_tests/test_status.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
```

Comment:
Can you move this test to `napari/utils/_tests/test_status.py`?

Reply:
ack, yes - I should have looked more closely at the organization.